providers/qedr/qelr_main.c

```diff
@@ -113,6 +113,14 @@ static const struct verbs_context_ops qelr_ctx_ops = {
 	.free_context = qelr_free_context,
 };
 
+static const struct verbs_context_ops qelr_ctx_roce_ops = {
+	.create_qp_ex = qelr_create_qp_ex,
+	.open_xrcd = qelr_open_xrcd,
+	.close_xrcd = qelr_close_xrcd,
+	.create_srq_ex = qelr_create_srq_ex,
+	.get_srq_num = qelr_get_srq_num,
+};
```

Comment:
Keep lists sorted

Reply:
> Keep lists sorted

Well..., In almost all providers this isn't kept sorted :-), someone forgot to do that ;-).

Anyway, fixed.